src/app/public/modules/infinite-scroll/infinite-scroll.component.ts

```diff
@@ -31,9 +31,15 @@ import {
   ]
```

Comment:
Should we have a description for the component like we have with the repeater item component? Maybe `Creates a container to display repeater items.`?

Reply:
I assume you meant for this comment to go on the `sky-repeater` ? I added your suggestion to it.

Reply:
Looks like I had my wires crossed and/or the screen shifted without my noticing. Regardless, your assumption was correct. Thanks!